### DISCUSSION TITLE
Version 6.0.0 no popup for Port 3000

### DISCUSSION THREAD

**Discussion by Mark-Muc** [benzino77/tasmocompiler#214] [General]
I tried the new Version to compile a shutter version.
First thing:
no browser popup for the Port 3000 (on edge and chrome):
so u have to add manually "3000-"betweem "htts//:" and the "compiler URL"
Second. 
the timeout of GitHub is very smal .. u have to hurry to build a new version .. enter you data quick.
Can u have a look into it.
THX
Mark

**Comment by benzino77** [comment]
Are you using gitpod?

**Reply by Mark-Muc** [reply]
how i can find out?
My page looks like this .. 
![Gidpod Code](https://user-images.githubusercontent.com/67410847/126030340-65078812-45b6-4c5c-81a7-d012191eabc0.png)

**Comment by benzino77** [comment]
You are using Gitpod to run TasmoCompiler.
I've discovered that Gitpod has changed it's settings:
![image](https://user-images.githubusercontent.com/15658235/126034110-e171d3b9-5425-43ec-9e37-cbc2e51b6dfc.png)

There is a need to change property `visibility` on `.gitpod.yml` in TasmoCompiler.

**Comment by benzino77** [comment]
It should be fixed now. Close all your current workspaces with TasmoCompiler on Gitpod and start a new one.

**Reply by Mark-Muc** [reply]
Thanks for looking into it. I have done like u said...
_...with TasmoCompiler on Gitpod and start a new one._
but unfortunately the same behavior ..
It say:
.... 

> Requirement already satisfied: urllib3<1.27,>=1.21.1 in /workspace/.pip-modules/lib/python3.7/site-packages (from requests==2.*->platformio) (1.26.6)
> Server started on port 3000

but no popup or hint!!
using: https://3000-red-...
works like expected and I can compile a version.
Regards
Mark

**Reply by benzino77** [reply]
I don't understand. Is it working for you or not?

Every modern browser is blocking popup windows. You have to explicitly allow to open popup window. It is reported by most browsers on the address/search bar:
Brave Browser:
![gitpod-brave](https://user-images.githubusercontent.com/15658235/126060776-cb841be3-9b8f-45e9-910f-79b6f8b1ff73.png)

Firefox:
![gitpod-firefox](https://user-images.githubusercontent.com/15658235/126060784-4700f28b-4a94-4785-b084-b574d1807d04.png)

Chrome:
![gitpod-chrome](https://user-images.githubusercontent.com/15658235/126060787-7fcddc37-bc42-4336-b598-f3870f19ef3e.png)

**Comment by bovirus** [comment]
@Mark-Muc

For me it works with Firefox and Chrome.

**Reply by Mark-Muc** [reply]
so i tested ,, the problem caused by a browser extension for MS Edge & Chrome called: 
**ClearURLs**
Größe: 2,3 MBGröße2,3 MBVersion: 1 . 2 1 . 0Version1.21.0
https://microsoftedge.microsoft.com/addons/detail/clearurls/mdkdmaickkfdekbjdoojfalpbkgaddei
btw. I run it on MS Edge (chromium; DEV edition)

after deactivated it .. the popup shows up like it was before .. thank u for your help .. maybe someone else run into the same problem!!
Regards 
Mark

**Reply by Mark-Muc** [reply]
so i now figured out what's the problem is / was .. if u disable the E-TagHeader 
![ETH Header](https://user-images.githubusercontent.com/67410847/126122953-b9b9c7dd-8c17-4a16-a1d5-6763bc0cf7e3.png)
 filtering it works also if the plugin ClearURLs is enabled
Mark
